test/unit/jira/__snapshots__/connect.test.ts.snap

```diff
@@ -0,0 +1,230 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
One of these snapshots is duplicated from the maintenance mode tests. Maybe it would make sense to combine the tests. But I'dnt worry too much hence we'll remove maintenance mode tests on the feature flag celanup.